src/QirRuntime/lib/Tracer/tracer.cpp

```diff
@@ -0,0 +1,305 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <cassert>
+#include <set>
+#include <sstream>
+
+#include "tracer.hpp"
+
+using namespace std;

```

Comment:
You write `using namespace std` but then you still write `std::` explicitly. Why?

Reply:
I don't have a rule of thumb. Mostly, if it's long type that stands out, like chrono::tah-dah-dah::tah-dah-dah, I omit 'std::' but if the typename is short or there is plenty of space on the line, I retain 'std::'... Just a personal readability adaptation, I guess. Don't know if it's part of our coding style or if it matters enough to be codifying in the style.